pkg/routing/localrouter.go

```diff
@@ -154,9 +148,7 @@ func (r *LocalRouter) statsWorker() {
 		}
 		// update every 10 seconds
 		<-time.After(statsUpdateInterval)
-		r.lock.Lock()
-		r.currentNode.Stats.UpdatedAt = time.Now().Unix()
-		r.lock.Unlock()
+		r.currentNode.UpdateNodeStats()
```

Comment:
Old code was only setting updated at, but changing it to do the full node stats update to make it consistent.